tests/FSharp.Compiler.ComponentTests/Conformance/LexicalFiltering/OffsideExceptions/RelaxWhitespace2.fs

```diff
@@ -3568,7 +3568,7 @@ and             ^a
          > () = 
             member inline __.X = ()
         with
-        static member inline Y = ()
+            static member inline Y = ()
```

Comment:
I updated this test. Without the additional indent in now produces a new indentation warning with the old rules and an error with the new ones.